sql/updates/world/master/2021_02_14_01_world_druid_infected_wounds.sql

```diff
@@ -0,0 +1,5 @@
+-- Add infected wounds
+DELETE FROM `spell_proc` WHERE `SpellId` IN (48484, 345208);
+INSERT INTO `spell_proc` (`SpellId`, `SchoolMask`, `SpellFamilyName`, `SpellFamilyMask0`, `SpellFamilyMask1`, `SpellFamilyMask2`, `SpellFamilyMask3`, `ProcFlags`, `SpellTypeMask`, `SpellPhaseMask`, `HitMask`, `AttributesMask`, `DisableEffectsMask`, `ProcsPerMinute`, `Chance` , `Cooldown`, `Charges`) VALUES
+(48484, 1, 7, 0x1000, 0, 0, 0, 0x10, 1, 1, 0x403, 0, 0, 0, 100, 0, 0),
+(345208, 1, 7, 0x800, 0x40, 0, 0, 0x10, 1, 1, 0x403, 0, 0, 0, 100, 0, 0);
```

Comment:
This content is redundant, duplicate of the last 2 lines in `9999_99_99_99_world_druid_guardian_spells.sql` (next file) and should be removed.

Reply:
I'm assuming there will be other comments btw, so I'll remove this file together with the other fixes to be.

Reply:
OK, fair enough. Sounds good to me.